src/platforms/web/runtime/transition-util.js

```diff
@@ -179,3 +179,24 @@ function getTimeout (delays: Array<string>, durations: Array<string>): number {
 function toMs (s: string): number {
   return Number(s.slice(0, -1)) * 1000
 }
+
+function parseDuration (value: any): number | typeof NaN | typeof undefined {
```

Comment:
`typeof NaN` is effectively `number` in flow-type. https://flowtype.org/try/#0G4QwTgBCBcEC4E8AOBTA9gMwgORNiAvBAIwBMAzEA